CHANGELOG.md

```diff
@@ -651,6 +651,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Bug Fixes
 
+* Generate valid xcconfig when target name includes spaces.
```

Comment:
the sentence part of the changelog entry should have two trailing spaces after `.` to force a line break